src/App.php

```diff
@@ -320,6 +321,98 @@ public function getRequest(): ServerRequestInterface
         return $this->request;
     }
 
+    /**
+     * Check if a specific GET parameter exists in the HTTP request.
+     */
+    public function hasRequestGetParam(string $key): bool
+    {
+        return $this->tryGetRequestGetParam($key) !== null;
+    }
+
+    /**
+     * Try to get the value of a specific GET parameter from the HTTP request.
+     */
+    public function tryGetRequestGetParam(string $key): ?string
+    {
+        return $this->request->getQueryParams()[$key] ?? null;
+    }
+
+    /**
+     * Get the value of a specific GET parameter from the HTTP request.
+     */
+    public function getRequestGetParam(string $key): string
```

Comment:
@abbadon1334 do you prefer `getRequestGetParam` over `getRequestQueryParam`?

or what about `getRequestQuery($k)`/`getRequestPost($k)`?

Reply:
laravel: `$request->query('name');`
slim: `$request->getQueryParam('page');`
Symfony: `$request->query->get('name');`

i think **Query** is the correct way of naming it, but i should leave **Param**.
So i think the best option is: `getRequestQueryParam($k)` and `getRequestPostParam($k)`

WDYT?